v2/docker-compose.yml

```diff
@@ -1,7 +1,33 @@
 version: "3.8"
 
 services:
-  
+          
+  orion:
```

Comment:
I received this warning, and the Orion is therefore not reachable 
`WARN@2024-07-30T08:16:43.378Z  mongoConnectionPool.cpp[306]: mongoc driver: <mongoc> <Last server removed from topology>`

Reply:
seems like I forgot to comment the replica set for the standalone version